src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1256,10 +1256,12 @@ internal static bool GetUnificationUseSiteDiagnosticRecursive(ref DiagnosticInfo
 
         #endregion
 
+#pragma warning disable CS0419 // Ambiguous reference in cref attribute
```

Comment:
📝 This can be resolved by placing a using alias directive at the top of the file. Both code and documentation comments prefer local aliases over other resolutions, and the alias will be resolved considering accessibility.